guacamole/src/main/frontend/src/app/auth/service/authenticationService.js

```diff
@@ -84,12 +84,7 @@ angular.module('auth').factory('authenticationService', ['$injector',
             return cachedResult;
 
         // Return explicit null if no auth data is currently stored
-        var data = localStorageService.getItem(AUTH_STORAGE_KEY);
-        if (!data)
-            return null;
-
-        // Update cache and return retrieved auth result
-        return (cachedResult = new AuthenticationResult(data));
+        return null;
```

Comment:
As this function will now only return the in-memory cached auth result provided via `setAuthenticationResult()`, we should note this in the function's docs, as well as the fact that `setAuthenticationResult()` will be called automatically upon page load.

Reply:
Will do